src/schrodinger/rdkit_extensions/convert.cpp

```diff
@@ -541,7 +541,9 @@ boost::shared_ptr<RDKit::RWMol> to_rdkit(const std::string& text,
                 // crash the Sketcher.  This #ifndef should be removed as part
                 // of SKETCH-2357.
                 Format::MRV,
-                Format::CDXML,
+        // Support for CDXML has been disabled
+        // due to build issues (SKETCH-2543)
+        // Format::CDXML,
```

Comment:
Will all of these changes will also be pushed in our internal repo?

Reply:
We should -- I'll ask for mgmt approval on the case

Reply:
Thanks. I really don't want sketcher and internal code out of sync.